### PR TITLE
Change Test-framework to xunit, and testing GiverController 

### PR DESCRIPTION
**Pull request by Sommerli**

### CODE REVIEW

File: GiEnJul.Test/Controllers/GiverControllerTest.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        public void ValidateInvalidGiverModels(Models.Giver model)
+        {
+            Assert.NotEmpty(ValidateModel(model));
+        }
+
+        [Fact]
+        public void ValidateValidGiverModel()
+        {
+            Assert.Empty(ValidateModel(new Models.Giver("Stavanger") { FullName = "Test Testesen", Email = "Test@Testesen.com", PhoneNumber = "12345678", MaxRecievers = 1 }));
+        }
```
Comment (Sommerli) [inline GiEnJul.Test/Controllers/GiverControllerTest.cs:62]: @RonHaland Should these test be seperated from GiverController to another Test-class?
They test the Dataannotations for Models.Giver.

Comment (RonHaland) [reply GiEnJul.Test/Controllers/GiverControllerTest.cs:62]: Yeah, probably

File: GiEnJul.Test/UnitTest1.cs [hunk 2]
```diff
@@ -1,18 +1,29 @@
-using NUnit.Framework;
+
```
Comment (RonHaland) [inline GiEnJul.Test/UnitTest1.cs:1]: could just remove this file

**Review by RonHaland — DISMISSED** [review]
Suggestion

**Review by TheseusDeus — APPROVED** [review]

**Pull request merged**